### PR TITLE
Reducer organization

### PR DESCRIPTION
**Pull request by ericheep**
Lots of changes.

Still more cleaning to do with index.js, thinking about moving the mouse listeners to their own listeners.js.

Also unsure about how much of the logic code belongs in the reducer files, it might be better later on to put that somewhere else in the future (if it even matters)?

### CODE REVIEW

File: src/reducers/amplitudesReducer.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const posterior = amplitudes.slice(endIndex + 1)
+
+  if (endIndex === startIndex)
+    return prior.concat([endValue]).concat(posterior)
+
+  const modified = amplitudes
+    .slice(startIndex, endIndex + 1)
+    .map(({ value }, idx) => amplitude((slope * idx) + startValue.value))
+
+  return prior.concat(modified).concat(posterior)
```
Comment (clarkenciel) [inline src/reducers/amplitudesReducer.js]: while we're doing cleanup what do you think of swapping this out for
```javascript
[...prior, ...modified, ...posterior]
```

File: src/actions/actions.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    position,
+  }),
+  setMouseUp: () => ({
+    type: 'SET_MOUSE_UP',
+  }),
+  setFrequency: frequency => ({
+    type: 'SET_FREQUENCY',
+    frequency,
+  })
+}
```
Comment (clarkenciel) [inline src/actions/actions.js]: what do you think of turning these into named consts and just exporting each one?
e.g.
```javascript
export const setMouseDown = () => ...;
export const setFrequency = frequency => ...;
// etc
```

**Review by clarkenciel — APPROVED** [review]
this looks great!

File: src/reducers/amplitudesReducer.js (hunk 1 above)
Comment (ericheep) [reply src/reducers/amplitudesReducer.js]: That looks more intuitive to me, I'm for it.

**Pull request merged**